nni/runtime/tuner_command_channel/channel.py

```diff
@@ -6,11 +6,19 @@
 """
 
 from __future__ import annotations
+import json
+from typing import Union
 
 __all__ = ['TunerCommandChannel']
 
 from .command_type import CommandType
 from .websocket import WebSocket
+from .semantic_command import ReportMetricData, UpdateSearchSpace
```

Comment:
Imported identifiers can be a tuple and a tuple can have line wrap.
`from x import (a, b, c, d, e, f, g)